thredds/values.yaml

```diff
@@ -70,7 +72,22 @@ config_files:
 storage_size: 5Gi
 
 storageClass:
-  storageClassName: ''
+  createClass: true
+  provisioner: efs.csi.aws.com
+  storageClassName: 'efs-pv'
+  parameters:
+    provisioningMode: efs-ap
+    fileSystemId: fs-51500hktech2023
+    directoryPerms: '700'
+    gidRangeStart: '1000'
+    gidRangeEnd: '2000'
+    basePath: '/data'
+  allowVolumeExpansion: true
+  volumeBindingMode: WaitForFirstConsumer
+  mountOptions:
+    - tls
+  annotations:
+    storageclass.kubernetes.io/is-default-class: 'true'
```

Comment:
Did you mean to commit this stuff? We don't want to have any cloud provider-specific values in this repo. This should probably go somewhere in the tethysportal-ciroh repo.

Reply:
Suggested default:

```suggestion
  createClass: false
  storageClassName: ''
```